src/components/Facets.tsx

```diff
@@ -29,17 +26,95 @@ export interface FacetsProps {
  * and {@link HierarchicalFacets} components can be used instead for more control over facet
  * configuration.
  *
+ * To override a single facet, use {@link StandardFacet}.
+ *
  * @param props - {@link FacetsProps}
  * @returns A React component for facets
  *
  * @public
  */
 export function Facets(props: FacetsProps) {
-  const { customCssClasses = {} } = props;
+  const { searchOnChange, children, customCssClasses = {} } = props;
+
+  const fieldIdToCustomFacetProps = new Map();
+  if (children) {
+    (Array.isArray(children) ? children : [children])
+      .forEach(child => fieldIdToCustomFacetProps.set(child.props.fieldId, child));
+  }
+
   return (
-    <div className={customCssClasses.facetsContainer}>
-      <StandardFacets/>
-      <NumericalFacets />
+    <div>
+      <FacetsProvider searchOnChange={searchOnChange} className={customCssClasses.facetsContainer}>
+        {facets => facets
+          .map((facet, i) => {
+            let facetType: FacetType = FacetType.STANDARD;
+            let facetProps: FacetProps = {
+              fieldId: facet.fieldId,
+              label: facet.displayName,
+            };
+            if (fieldIdToCustomFacetProps.has(facet.fieldId)) {
+              const customFacetElement: ReactElement = fieldIdToCustomFacetProps.get(facet.fieldId);
+              facetProps = customFacetElement.props;
+              facetType = getFacetTypeFromReactElementType(
+                (typeof customFacetElement.type === 'function') ? customFacetElement.type.name : '');
+            } else {
+              if (isStringFacet(facet)) {
+                facetType = FacetType.STANDARD;
+              } else if (isNumericalFacet(facet)) {
+                facetType = FacetType.NUMERICAL;
+              }
+            }
+
+            let facetComponent: ReactElement;
+            switch (facetType) {
+              case FacetType.NUMERICAL:
+                facetComponent = <div/>;
```

Comment:
is it safe to use an empty fragment here? `<></>` it feels weird to insert something into the dom

Reply:
also do we need to handle the hierarchical case?

Reply:
It's just a placeholder till we implement the NumericalFacet (which I'm doing right now)

Reply:
I imagine we'll eventually add the hierarchical case. But I want to keep the behavior of Facets to be the same as of now